i18n/en.json

```diff
@@ -66,5 +66,10 @@
   "check all": "Check all that apply",
   "User settings": "User settings",
   "Notification settings": "Notification settings",
-  "Logout": "Log out"
+  "Logout": "Log out",
+  "Create account": "Create account",
+  "go back": "Go back",
+  "Sign up": "Sign up",
+  "enter name": "Please enter your name",
+  "name": "Name"
```

Comment:
Do you think we need to make similar edits to `es.json` and `nl.json`? I haven't tested running the app with those other languages, so I'm not sure if the app will crash if those values are missing.

Reply:
You're very correct! Thanks